pkgs/by-name/gg/gg-jj/package.nix

```diff
@@ -33,10 +33,19 @@ rustPlatform.buildRustPackage (finalAttrs: {
   cargoHash = "sha256-AdatJNDqIoRHfaf81iFhOs2JGLIxy7agFJj96bFPj00=";
 
   npmDeps = fetchNpmDeps {
-    inherit (finalAttrs) pname version src;
-    hash = "sha256-izCl3pE15ocEGYOYCUR1iTR+82nDB06Ed4YOGRGByfI=";
+    inherit (finalAttrs)
+      pname
+      version
+      src
+      patches
+      ;
+    hash = "sha256-ehXGLpCCN+BNqtwjEatcfR0kQHj5WOofTDR5mLSVW0U=";
   };
 
+  patches = [
+    ./update-tauri.patch
```

Comment:
Can you link to your upstream patch submission?

Reply:
Whoops, sorry about that! It's https://github.com/gulbanana/gg/pull/68, I removed the loose patch file in this one.